.github/workflows/build_dev_documentation.yml

```diff
@@ -48,7 +48,7 @@ jobs:
           cd ..
 
           cd optimum
-          pip install .[dev,onnxruntime,intel]
+          pip install -e .[dev,onnxruntime,intel]
```

Comment:
Sorry, I am not very knowledgeable on the subject, but why do we need to install the packages in development mode?

Reply:
Sorry, that was me trying to debug the `doc-builder` in the GitHub actions - it's removed now!